### PR TITLE
Authentication plugin release news

### PR DESCRIPTION
**Pull request by burzum**

### CODE REVIEW

File: 2017/01/03/cakephp_psr7_middleware_authentication.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Where is the code? How do I install it?
+---------------------------------------
+
+Check `the official repository<https://github.com/cakephp/authentication>`_ of
+the plugin and install it via composer as usual.
+
+  ``composer require cakephp/authentication``
+
+Pay attention to the version constraints! Until CakePHP 3.4 is released it will
+require the 3.next branch!
```
Comment (dereuromark) [inline 2017/01/03/cakephp_psr7_middleware_authentication.rst]: Should this include the tip on how?

    "3.next as 3.4"

Most wouldn't know probably.

File: 2017/01/03/cakephp_psr7_middleware_authentication.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
+CakePHP PSR7 Middleware Authentication Released
+===============================================
+
+Introduction and background
+---------------------------
+
+Let’s start with some background story on this topic. One of the parts of the
+CakePHP framework I like the less is the authentication and authorization.
+There are a few reasons why. For example the authentication and authorization.
```
Comment (antograssiot) [inline 2017/01/03/cakephp_psr7_middleware_authentication.rst]: Is this  joke ? I quite don't get the intention of this sentence...

Comment (dereuromark) [inline 2017/01/03/cakephp_psr7_middleware_authentication.rst]: It is, even though it should read "the least " ;)

File: 2017/01/03/cakephp_psr7_middleware_authentication.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
+CakePHP PSR7 Middleware Authentication Released
+===============================================
+
+Introduction and background
+---------------------------
```
Comment (markstory) [inline 2017/01/03/cakephp_psr7_middleware_authentication.rst]: I would remove this heading it isn't necessary as nothing precedes it.

**Pull request merged**